crates/http-api-bindings/src/embedding/mod.rs

```diff
@@ -25,7 +27,14 @@ pub async fn create(config: &HttpModelConfig) -> Arc<dyn Embedding> {
             Arc::new(engine)
         }
         "ollama/embedding" => ollama_api_bindings::create_embedding(config).await,
-
+        "voyage/embedding" => {
+            let engine = VoyageEmbeddingEngine::create(
+                &config.api_endpoint,
+                config.model_name.as_deref().unwrap_or_default(),
```

Comment:
voyage requires model_name to be set - generate a panic (e.g `expect`) if it's not set

Reply:
Or should we choose a model for user in the case? such as `voyage-code-2`?

Reply:
preferring explicity in cases like this - let's throw an error if it's not set